db/migrate/20230404092416_create_likes.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :author_id
+      t.integer :post_id
```

Comment:
- [ ] Great job but I think you should reference  here to the author_id and post_id not to add a new column in the table just reference to those tables 🔥🤩🙏🏻to be as  foreign_keys 💯💯

## It would be somthing like this ⬇⬇

```ruby
  t.references :user, foreign_key: true 
    t.references :post, foreign_key: true
```